src/modules/Dashboard/api/api.ts

```diff
@@ -603,6 +606,17 @@ export const getAssessmentApi = ({ appletId, answerId }: AssessmentReview, signa
     },
   );
 
+export const getFlowAssessmentApi = (
+  { appletId, submitId }: AssessmentFlowReview,
```

Comment:
```suggestion
  { appletId, submitId }: AssessmentFlowReviewParams,
```

Reply:
Unfortunately not it's just a suggestion, but we can also discuss this inside of the team.